eclipse-scout-core/src/table/columns/ColumnModel.ts

```diff
@@ -46,6 +46,14 @@ export interface ColumnModel<TValue = string> extends ObjectModel<Column<TValue>
    */
   index?: number;
 
+  /**
```

Comment:
Maybe add "This is mainly used to restore the selection when rows are replaced using {@link table.replaceRows}"